src/util/settingsUtils.js

```diff
@@ -65,6 +65,23 @@ export function settingEnabled(data, enabledKey, startTimeKey, endTimeKey) {
 	return enabled && isWithinInterval(new Date(), { start: startTime, end: endTime });
```

Comment:
This could be refactored to use the new function below and avoid repeating code. So this return would read something like `return enabled && settingWithinDateRange(foo, bar, blah)`